TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -413,13 +422,30 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
 
                             // If an alphanumeric literal that is to be continued on the next line has as its last character a quotation mark in column 72, 
                             // the continuation line must start with two consecutive quotation marks.
+                            // In ColumnsLayout.CobolReferenceFormat we just check that the quotation mark is the last character of the line.
                             if (lastTokenOfConcatenatedLineSoFar.HasClosingDelimiter)
                             {
-                                if ((startOfContinuationIndex + 1) > lastIndex || line[startOfContinuationIndex + 1] != lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter)
+                                if (format == ColumnsLayout.CobolReferenceFormat 
+                                    ? (((startOfContinuationIndex + 1) <= lastIndex && line[startOfContinuationIndex + 1] == lastTokenOfConcatenatedLineSoFar.ExpectedClosingDelimiter) &&
+                                    (lastTokenOfConcatenatedLineSoFar.EndColumn + 8) != 73)
```

Comment:
I did not understand that part but I think we should avoid hardcoded constants. Does comparing `TextArea.EndColum` work ? Same thing for the `8` value, it should be hidden somewhere in the text area definitions.